src/fhir-tool-core/Operations/GenerateDocumentReferenceOption.cs

```diff
@@ -0,0 +1,79 @@
+﻿extern alias R4;
+using CommandLine;
+using FhirTool.Core.FhirWrappers;
+using R4::Hl7.Fhir.Model;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Threading.Tasks;
+using System.Linq;
+using System.IO;
+
+namespace FhirTool.Core.Operations
+{
+    [Verb("generate-documentreference", HelpText = "Generate documentreference file")]
+    public class GenerateDocumentReferenceOptions 
+    {
+        [Option('i', "id", HelpText = "Documentreference id", Required = true)]
+        public string Id { get; set; }
+
+        [Option('c', "contentType", HelpText = "contentType for attachment", Required = true)]
+        public string ContentType { get; set; }
+
+        [Option('p', HelpText = "Save documentreference to path", Required = true)]
```

Comment:
Som over, endre short argument til 'o' og argument til 'out': `[Option('o', "out", HelpText = "Out path for Binary", Required = true)]` og navn på property tilsvarende OutPath